src/styles/main.scss

```diff
@@ -74,10 +74,6 @@ $background-color: #f9f9f9;
     }
   }
 
-  // .edge-overlay-path, .edge-path {
-  //   pointer-events: none;
-  // }
```

Comment:
This code was initially checking if pointer-events could fix the arrow issues when entering nodes, but I later decided to dynamically add these styles using react when the arrows were being dragged instead of adding them using classes.